Task10_Auth/src/main/java/LoadProperties.java

```diff
@@ -0,0 +1,28 @@
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+public class LoadProperties {
```

Comment:
It doesn't make sense to create objects of this class and read properties every time. That's why remove constructor  and move all code to static block:
`static {
//code from current ctor
}`
And add some method like:
`public static String get(String key) {...}`

Reply:
done